submissions/Oleksii_Drohachov/HTML_CSS_PopUp/index.html

```diff
@@ -0,0 +1,228 @@
+              >My lovely course</a
+            >
+          </li>
+          <li class="nav__item">
+            <a class="nav_link" href="https://t.me/+kI9jsoLH2rQ4YzVi"
+              >My Kottans in TG</a
+            >
+          </li>
+          <li class="nav__item__menu">
+            <label for="first"></label>
+            <input id="first" class="first" type="checkbox" name="first" />
+            <div class="main__wrapper">
+              <div class="menu__list__main">
+                <div class="menu__item__main" tabindex="0">
+                  <div class="item_box">
+                    <div><img src="icons/apps.png" alt="icon logo" /></div>
+                    Strange BMW logo
+                  </div>
+                </div>
+                <div class="menu__item__main" tabindex="0">
+                  <div class="item_box">
+                    <div><img src="icons/bell.png" alt="icon logo" /></div>
+                    Ringing bell
+                  </div>
+                </div>
+                <div class="menu__item__main" tabindex="0">
+                  <div class="item_box">
+                    <div><img src="icons/blogs.png" alt="icon logo" /></div>
+                    List of tasks
+                  </div>
+                </div>
+                <div class="menu__item__main" tabindex="0">
+                  <div class="item_box">
+                    <div><img src="icons/calendar.png" alt="icon logo" /></div>
+                    Studying plan
+                  </div>
+                </div>
+                <div class="menu__item__main" tabindex="0">
+                  <div class="item_box">
+                    <div><img src="icons/chat.png" alt="icon logo" /></div>
+                    Kottans chat
+                  </div>
+                </div>
+                <div class="menu__item__main" tabindex="0">
+                  <div class="item_box">
+                    <div><img src="icons/maps.png" alt="icon logo" /></div>
+                    Treasure map
+                  </div>
+                </div>
+                <div class="menu__item__main" tabindex="0">
+                  <div class="item_box">
+                    <div><img src="icons/mail.png" alt="icon logo" /></div>
+                    Msgs from mentor
+                  </div>
+                </div>
+                <div class="menu__item__main" tabindex="0">
+                  <div class="item_box">
+                    <div><img src="icons/files.png" alt="icon logo" /></div>
+                    Project folder
+                  </div>
+                </div>
+                <div class="menu__item__main" tabindex="0">
+                  <div class="item_box">
+                    <div><img src="icons/photos.png" alt="icon logo" /></div>
+                    Screenshots
+                  </div>
+                </div>
+              </div>
+
+              <label class="second_label" for="second">Even more icons !</label>
```

Comment:
Hide this button when the second part of popup is open
![image](https://user-images.githubusercontent.com/16196199/183967773-b587e9de-5996-4c1f-aeba-b94b30191d3c.png)